src/main/java/util/number/SecureRandomNumberGenerator.java

```diff
@@ -0,0 +1,17 @@
+package util.number;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class SecureRandomNumberGenerator {
+    private static final int MAX_NUMBER = 9;
+
+    public static int getRandomNumber() {
+        int random = 0;
+        try {
+            random = SecureRandom.getInstanceStrong().nextInt(MAX_NUMBER);
+        } catch (NoSuchAlgorithmException ignored) {
+        }
+        return random;
```

Comment:
위 코드는 아래 코드로 수정될 수 있겠네요.
```suggestion
        try {
            return SecureRandom.getInstanceStrong().nextInt(MAX_NUMBER);
        } catch (NoSuchAlgorithmException ignored) {
            return 0;
        }
```